examples/iterative-refinement/data/A.mtx

```diff
@@ -0,0 +1,114 @@
+%%MatrixMarket matrix coordinate integer symmetric
```

Comment:
Maybe we can move this from all the specific examples directories to a common directory. Now we just duplicate this matrix everywhere.